keda/values.yaml

```diff
@@ -2,20 +2,31 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
+global:
+  image:
+    # -- Global image registry of KEDA components
+    registry: null
+
 image:
   keda:
+    # -- Image registry of KEDA operator
+    registry: ghcr.io
     # -- Image name of KEDA operator
-    repository: ghcr.io/kedacore/keda
+    repository: kedacore/keda
```

Comment:
These are breaking changes, we'll need to revert it and find another way to not break people

Reply:
I'm afraid that doing in another way would diverge il much with what all the other charts in the wild do though. I'm fine if it makes out in 2.13